internal/cmd/server.go

```diff
@@ -134,6 +138,12 @@ The kernel is used to run long running processes like shells and interacting wit
 			if enableRunner {
 				runnerv1.RegisterRunnerServiceServer(server, runner.NewRunnerService(logger))
 			}
+
+			healthcheck := health.NewServer()
+			healthgrpc.RegisterHealthServer(server, healthcheck)
+			// Setting SERVING for the whole system.
+			healthcheck.SetServingStatus(system, healthgrpc.HealthCheckResponse_SERVING)
```

Comment:
```suggestion
			healthcheck.SetServingStatus("", healthgrpc.HealthCheckResponse_SERVING)
```

Would prefer this inlined if that's alright!